core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.component.html

```diff
@@ -199,6 +200,17 @@
   .oppia-topic-editor-save-icon {
       padding-left: 5px;
   }
+
+  @media screen and (max-width: 991px) {
+    .oppia-editor-publish-button-container {
+      margin-right: 10px; margin-top: 8px;
```

Comment:
Please move these styles into 2 separate lines.

Reply:
Made the changes. @Vir-8